firmware/t3b1/releases.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Why create this file? This device does not exist, and I don't think we should have it here if it's not being utilized.

Reply:
We thought with @komret that it'd be convenient to have it prepared for future. But I agree its useless, I shall remove it :slightly_smiling_face:

Reply:
Yeah, there is no real guarantee that this model will ever exist, so there's no point in having it here before we need it.